### PR TITLE
Done with delete application modal

### PR DESCRIPTION
**Pull request by haophn01**
Delete Application Modal Prompt is done
Screenshot 2024-11-17 at 12 39 37 PM

### CODE REVIEW

File: client/src/ui/track/ApplicationDeleteModal.tsx [hunk 1]
```diff
@@ -56,14 +56,58 @@ export default function ApplicationDelete({
...
-        <Typography id="simple-modal-title">
+        <Typography id="simple-modal-title" variant="h6">
           Are you sure you want to delete this application?
         </Typography>
         <Typography id="simple-modal-description">
           This action cannot be undone.
         </Typography>
-        <Button onClick={handleClose}>Cancel</Button>
-        <Button onClick={handleDelete}>Delete</Button> 
+        <Box
```
Comment (MekhiHart) [inline client/src/ui/track/ApplicationDeleteModal.tsx:65]: Can we add padding to the modal so the content is not as close to the border. It feels claustrophobic with the lack of spacing.

**Review by MekhiHart — CHANGES_REQUESTED** [review]
Can the modal buttons have the same colors as the buttons in our app remain consistency

Reference from PrivateMessageModal:
image

File: client/src/ui/track/ApplicationDeleteModal.tsx (hunk 1 above)
Comment (haophn01) [reply client/src/ui/track/ApplicationDeleteModal.tsx:73]: Fixed!

**Review by MekhiHart — APPROVED** [review]
nice

**Pull request merged**